# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: supply-chain/config.toml
# Commit: 1d5a44ae7207
# Full Hash: 1d5a44ae72071f079dcb00b62419c2ddd3c9f12c
# Author: Norisz Fay <nfay@mozilla.com>
# Date: 2023-01-27 21:11 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing build bustages on DecoderData.cpp CLOSED TREE
# ==============================================================================

diff -r 83609f9575ac -r 1d5a44ae7207 supply-chain/config.toml
--- a/supply-chain/config.toml	Fri Jan 27 19:06:16 2023 +0000
+++ b/supply-chain/config.toml	Fri Jan 27 21:11:42 2023 +0200
@@ -671,6 +671,10 @@
 version = "1.0.2"
 criteria = "safe-to-run"
 
+[[exemptions.humantime]]
+version = "2.1.0"
+criteria = "safe-to-deploy"
+
 [[exemptions.hyper]]
 version = "0.14.19"
 criteria = "safe-to-run"